Solutions/ESI - Exchange Security Config and Audit Insights/Analytic Rules/CriticalCmdletsUsageDetection.yaml

```diff
@@ -0,0 +1,57 @@
+id: 5170c3c4-b8c9-485c-910d-a21d965ee181
+name: VIP Mailbox manipulation
+description: |
+  'High important Cmdlet executed on a VIP Mailbox'
+requiredDataConnectors:
+  - connectorId: ESI-ExchangeAdminAuditLogEvents
+    dataTypes:
+      - Event
+severity: Low
+queryFrequency: 30m
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+status: Available
+tactics:
+  - Exfiltration
+  - Persistence
+  - Collection
+relevantTechniques:
+  - T1020
+  - T1098
+  - T1114
+query: |
+  ExchangeAdminAuditLogs
```

Comment:
The query description says critical cmdlet used on VIP mailboxes - wondering if we need another where condition that matches for those critical commands.

Reply:
For the moment, my team think it's not revelant to have an alert on critical cmdlets if it's not linked to a VIP as this will generate a lot of incident without important things.